.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
```

Comment:
why github actions versus TravisCI?

Reply:
i'm not sure we want to migrate any of our other integration tests over to github actions at this time, but i'd be curious to see if github actions performs any better than travis.